tests/bin/radical-pilot-test-verify

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import requests
+import sys
+
+from datetime import datetime
+from argparse import ArgumentParser
+
+
+# ------------------------------------------------------------------------------
+#
+def main():
+
+    parser = ArgumentParser()
+    parser.add_argument('--yaml', '-y', type=str, required=True,
+                        help='Name of workflow YAML file')
+    parser.add_argument('--days', '-d', type=int, required=True,
+                        help='Number of days from last triggered check')
+    args = parser.parse_args()
+
+    url = 'https://api.github.com/repos/radical-cybertools/radical.pilot' + \
+          '/actions/workflows/%s.yml/runs?event=repository_dispatch' % args.yaml
+    resp = requests.get(url)
+    workflow_json = resp.json()
+
+    if workflow_json['total_count'] > 0:
+
+        last_run = workflow_json['workflow_runs'][0]
+        finished = last_run['updated_at']
+        status   = last_run['conclusion']
+
+        now      = datetime.now()
+        last     = datetime.strptime(finished, '%Y-%m-%dT%H:%M:%SZ')
+        diff     = now - last
+
+        if status == 'success' and diff.days > args.days:
+            sys.exit(1)
+        else:
```

Comment:
this implies:
```py
if status == failed:
     exit(0)
```
Is that intentional?

Reply:
Yes, this is intentional. I wanted to have the github action fail. For this to happen, an executable needs to exit with a code different from 0.